presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastoreConfig.java

```diff
@@ -0,0 +1,75 @@
+package com.facebook.presto.hive.metastore.thrift;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+
+import javax.validation.constraints.NotNull;
+
+public class ThriftHiveMetastoreConfig {
+
+  private final String PREFIX = "metastore";
+
+  private int maxRetries = 10;
+  private double backoffScaleFactor = 2.0;
+  private Duration minBackoffDelay = Duration.valueOf("1s");
```

Comment:
Use the direct constructor rather than the string parsing version (the Hive `RetryDriver` code where you copied this from is wrong):
```java
private Duration minBackoffDelay = new Duration(1, SECONDS);
private Duration maxBackoffDelay = new Duration(30, SECONDS);
private Duration maxRetryTime = new Duration(30, SECONDS);

Reply:
Sure. 
And I find that from the RetryDriver, default value for both `minBackoffDelay` and `maxBackoffDelay` is 1 second, and the default value for `maxRetryTime` is 30s. I will change my code;